script/foreman-debug

```diff
@@ -347,7 +363,6 @@ add_files /etc/{sysconfig,default}/foreman
 add_files /etc/{sysconfig,default}/libvirt*
 add_files /etc/sysconfig/pgsql
 add_files /var/lib/pgsql/data/pg_log/*
-add_cmd "foreman-rake plugin:list" "plugin_list"
```

Comment:
As much as I like this removal, I think we should keep this in separate commit and new issue. I know it looks bureaucratic, but it's very useful later to apply the policy of one issue per commit.

Reply:
There's no time, removed.